frame/example-offchain-worker/src/lib.rs

```diff
@@ -124,6 +129,13 @@ decl_module! {
 		fn offchain_worker(block_number: T::BlockNumber) {
 			debug::info!("[OCW] Hello World from offchain workers!");
 
+			let check_result = Self::check_if_should_proceed(block_number);
+
+			if let Err(e) = check_result {
```

Comment:
Result means the possibility of errors usually. Here this is the base logic so "should_proceed" as a boolean is more appropriate.